.github/workflows/test.yml

```diff
@@ -194,52 +158,13 @@ jobs:
       run: |
         ccache -M 128M && ccache -c
 
-    - name: Cache ttk-paraview package
-      uses: actions/cache@v2
-      id: cache-ttk-paraview
-      with:
-        path: "ttk-paraview/build/ttk-paraview.tar.gz"
-        key: ttk-paraview-macos
-
-    - uses: actions/checkout@v2
-      if: steps.cache-ttk-paraview.outputs.cache-hit != 'true'
-      with:
-        repository: "topology-tool-kit/ttk-paraview"
-        ref: "5.9.0"
-        path: "ttk-paraview"
-      name: Checkout TTK-ParaView source code
-
-    - name: Create & configure ParaView build directory
-      if: steps.cache-ttk-paraview.outputs.cache-hit != 'true'
-      run: |
-        # switch to Xcode 11 since Xcode 12 breaks the ParaView build
-        sudo xcode-select -s "/Applications/Xcode_11.7.app"
-        cd ttk-paraview
-        mkdir build && cd build
-        cmake \
-          -DCMAKE_BUILD_TYPE=Release \
-          -DPARAVIEW_USE_QT=OFF \
-          -DPython3_ROOT_DIR=$(brew --prefix python) \
-          -GNinja \
-          ..
-
-    - name: Build ParaView
-      if: steps.cache-ttk-paraview.outputs.cache-hit != 'true'
-      run: |
-        cd ttk-paraview/build
-        cmake --build . --parallel
-        sudo cmake --build . --target install
-
-    - name: Create ParaView package
-      if: steps.cache-ttk-paraview.outputs.cache-hit != 'true'
+    - name: Fetch TTK-ParaView headless macOS binary archive
       run: |
-        cd ttk-paraview/build
-        sudo cpack -G TGZ
+        wget https://github.com/pierre-guillou/paraview-ttk/releases/download/5.9.1-headless/ttk-paraview-headless.tar.gz
```

Comment:
this asset should be available on `https://github.com/topology-tool-kit/ttk-paraview` instead.
thanks PRing this repository accordingly.